lib/salus/scanners/report_node_modules.rb

```diff
@@ -103,6 +110,36 @@ def record_dependencies_from_package_lock_json
       end
     end
 
+    def record_dependencies_from_bower_json
+      packages = JSON.parse(@repository.bower_json)
+      # Record if custom registry is defined in .bowerrc
+      # Supports single and multiple registry -
+      # {"registry": "https://registry.bower.io"}
+      # {"registry": {"search": ["http://localhost:9000","https://registry.bower.io"] }}
+      sources = '<bower manager default>'
+      if @repository.bowerrc_present?
+        file = JSON.parse(@repository.bowerrc)
+        if file.key?("registry")
+          is_nested = file["registry"].is_a? String
+          sources = is_nested ? file["registry"] : file["registry"]["search"].join(',')
+        end
+      end
+
+      # Record dependencies, version from bower.json
+      packages['dependencies']&.each do |name, version|
+        record_node_module(
+          name: name,
+          version: version,
+          source: sources,
+          dependency_file: 'bower.json'
+        )
+      end
+    rescue JSON::ParserError => e
```

Comment:
The `rescue JSON::ParseError` should only be applied to the two `JSON.parse(...)`, like
```ruby
begin
  file.JSON.parse(@repository.bower_json)
rescue JSON::ParseError ...
```